VideoLocker/src/main/java/org/edx/mobile/view/CourseBaseActivity.java

```diff
@@ -134,7 +105,6 @@ protected void onDestroy() {
         isDestroyed = true;
```

Comment:
I think this was added by you at some point. Just curious, what was its purpose ?